.github/workflows/publish.yml

```diff
@@ -1,11 +1,12 @@
 # see https://github.com/viamrobotics/upload-module for help
+name: Publish Module
+
 on:
-    push:
-    release:
-      types: [published]
```

Comment:
I'm not sure if this is needed

Reply:
@acmorrow actually, we might want this. might simplify your release workflow a bit.

It wouldn't be checking for version tags though using the regex that checks if it starts with `vX.Y.Z`